ipaplatform/redhat/authconfig.py

```diff
@@ -2,7 +2,7 @@
 #          Alexander Bokovoy <abokovoy@redhat.com>
 #          Tomas Babej <tbabej@redhat.com>
 #
-# Copyright (C) 2007-2014  Red Hat
+# Copyright (C) 2007-2016  Red Hat, Inc.
```

Comment:
If you're changing licence, please change it to the new format (see, e.g., ipaclient/install/client.py).